.gitignore

```diff
@@ -4,6 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
+public/env-config.js
```

Comment:
It seems that this file is included in the repo, but at the same time ignored here. Is this an accident or is the goal to not override the defaults?

Reply:
Accident, I thought I made also the removal.